docs/yaml_docs/pipeline_visualization_yml.md

```diff
@@ -0,0 +1,161 @@
+
+<span class="parameter">condaenv</span> `String` (Path)<br>
+    Path to conda environment that should be used to run panpipes.
+    Leave blank if running native or your cluster automatically inherits the login node environment
+
+## Loading and merging data options
+### Data format
+
+<span class="parameter">sample_prefix</span> `String`, Mandatory parameter, Default: test<br>
+Prefix for the sample that comes out of the filtering/ preprocessing steps of the workflow.
+
+<span class="parameter">mudata_obj</span> `String`, Mandatory parameter <br>
+ Path to the output file from preprocessing (e.g. `../vis/test.h5mu`).
+ Ensure that the submission file is in the right format and that the correct path is provided.
+
+<span class="parameter">modalities</span><br>
+<span class="parameter">rna</span> `Boolean`, Default: True <br>
+<span class="parameter">prot</span> `Boolean`, Default: True <br>
+<span class="parameter">atac</span> `Boolean`, Default: False <br>
+<span class="parameter">rep</span> `Boolean`, Default: True <br>
+<span class="parameter">multimodal</span> `Boolean`, Default: True <br>
+Set the modalities to True or False depending on what is present in the mudata_obj
+
+<span class="parameter">grouping_vars</span> `String`, Default: sample_id  rna:leiden_res0.6 <br>
+On dot plots and bar plots, grouping vars are used to group other features (for categorical, continuous, and feature plots).
+
+## Plot Markers 
+  
+The csv filed can be specified in the `vis` configuration file as follows:
+
+pipeline_vis config file: (pipeline.yml)
+
+```yaml
+# the full list will be plotted in dot plots and matrix plots, one plot per group
+full:
+ - long_file1.csv
+ - long_file2.csv
+# the shorter list will be plotted on umaps as well as other plot types, one plot per group
+minimal:
+ - short_file1.csv
+
+```
+<span class="parameter">custom_markers</span><br>
+  - <span class="parameter">files</span><br>
+  
+    - <span class="parameter">full:</span><br>
+The full list will be plotted in dot plots and matrix plots, with one plot per group. 
+
+     - <span class="parameter">minimal:</span><br>
+The shorter list will be plotted on umaps as well as other plot types, with one plot per group. 
+
+
+- <span class="parameter">paired_scatter:</span>`String`, Default:  <br>
+  Where different normalization exists in a modality, choose which one to use, set X or leave blank to use the mdata[mod].X assay. 
+
+- <span class="parameter">layers:</span><br>
+  - <span class="parameter">rna:</span>`String`, Default: logged_counts<br>
+  - <span class="parameter">prot:</span>`String`, Default: clr<br> 
+  - <span class="parameter">atac:</span>`String`, Default: signac_norm<br> 
+Check [gene_list_format.md](https://github.com/DendrouLab/panpipes/edit/clustering_g/docs/usage/gene_list_format.md) for Plot marker csv format instructions. 
```

Comment:
Can you put this right after the header, as its easier for the user to spot then,